Makefile.cross-compiles

```diff
@@ -19,7 +19,15 @@ app:
 		env CGO_ENABLED=0 GOOS=$${os} GOARCH=$${arch} GOMIPS=$${gomips} go build -trimpath -ldflags "$(LDFLAGS)" -o ./release/frps_$${target_suffix} ./cmd/frps;\
 		echo "Build $${os}-$${arch} done";\
 	)
-	@mv ./release/frpc_windows_386 ./release/frpc_windows_386.exe
-	@mv ./release/frps_windows_386 ./release/frps_windows_386.exe
-	@mv ./release/frpc_windows_amd64 ./release/frpc_windows_amd64.exe
-	@mv ./release/frps_windows_amd64 ./release/frps_windows_amd64.exe
+	@if [ -f ./release/frpc_windows_386 ]; then \
```

Comment:
Very minor change that allows excluding the windows binary (useful while testing)